tests/integration/test_format_txts_e2e.py

```
@@ -0,0 +1,266 @@
+        "-n", "Robert",
+        "-d", "2024-01-01",
+        "-s", "0.5",
+    ]
+    
+    # Mock the paths and environment
+    monkeypatch.setattr("format_txts.CONTACT_STORE_FILE", env["contacts_file"])
+    monkeypatch.setattr("format_txts.OUTPUT_DIR", env["output_dir"])
+    monkeypatch.setattr("pathlib.Path.home", lambda: env["export_dir"].parent)
+    
+    # Mock the run_imessage_exporter function to return our test export directory
+    async def mock_run_imessage_exporter(*args, **kwargs):
```

Ruff (0.14.2)

Missing return type annotation for private function `mock_run_imessage_exporter`

(ANN202)